build.sbt

```diff
@@ -78,6 +84,7 @@ val commonLib = (project in file(s"$appsFolder/common-lib"))
   .settings(
     commonSettings,
     libraryDependencies ++= Seq(
+      ws,
```

Comment:
This is defined for play projects in https://github.com/guardian/typerighter/blob/jsh/add-hmac/build.sbt#L100, do we need it in the commonLib?

Reply:
Thanks for spotting! fb3989918bb29fae1e506737fc94d408c66acdac